dev.yml

```diff
@@ -7,7 +7,7 @@ up:
     - openssl
     - sqlite
   - ruby:
-      version: 2.2.3p172-shopify
+      version: 2.3.3
```

Comment:
- People would use 2.3 feature in dev just to be surprised by the CI breaking when they submit their PR
- What advantage would that give us?